core/src/main/java/com/linecorp/armeria/common/logging/RequestLogBuilder.java

```diff
@@ -294,6 +295,15 @@ void session(@Nullable Channel channel, SessionProtocol sessionProtocol, @Nullab
      */
     void responseTrailers(HttpHeaders responseTrailers);
 
+    /**
+     * Sets the {@link RequestLog#responseCause()}.
+     * This method should be used when an exception is raised, and you want to convert the exception into a
+     * normal response such as {@link CloseWebSocketFrame}.
+     * If you want to end the response right away when an exception is raised,
+     * please use {@link #endResponse(Throwable)}.
```

Comment:
I kind of feel like this Javadoc is too specific to WebSocket. Would you mind rewriting this so it explains what this method does without using WebSocket as an example?